lib/llm/providers/anthropic.rb

```diff
@@ -61,5 +62,9 @@ def auth(req)
     def response_parser
       LLM::Anthropic::ResponseParser
     end
+
+    def error_response_parser
```

Comment:
@antaz Maybe this could just be `error_parser` ? The wording of this one feels strange.

Reply:
I settled on `ErrorHandler`. Wdyt ?

Reply:
@0x1eef `ErrorHandler` sounds fine.